js/app.js

```diff
@@ -0,0 +1,91 @@
+// sign-in input validator
+
+const mailInput = document.querySelector(".sign-up__form--input input");
```

Comment:
Bardzo rzadko łapiemy elementy po klasach, a jeszcze rzadziej po tagach. Dlaczego? Struktura HTML ma to do siebie, że może łatwo ulec zmianie i wtedy Twój kod JSowy albo nie zadziała, albo doprowadzi do bugów na stronie.

Elementy, które chcesz złapać z poziomu JSa łap albo poprzez:
- `id`, szeroko stosowane rozwiązanie
- `data-attribute`, który nadasz. To jest nawet lepszą opcją, bo developer wchodzący po Tobie na projekt łatwo zauważy, że konkretne elementy muszą mieć konkretne `data-attr`, żeby interakcje się nie sypały. Temu konkretnemu elementowi mógłbyś nadać np. attrybut `data-signup-input` i po tym go tutaj łapać